models/questions.py

```diff
@@ -15,10 +15,10 @@
 
 # In SQL we can manually add the constraint
 # alter table questions add constraint name_bc_unique UNIQUE(name, base_course);
-try:
-    db.executesql('''alter table questions add constraint name_bc_unique UNIQUE(name, base_course)''')
-except:
-    pass
+# try:
+#     db.executesql('''alter table questions add constraint name_bc_unique UNIQUE(name, base_course)''')
+# except:
+#     pass
```

Comment:
Hmm, well we need to do something other than this, because it breaks things when there's another migration to be done.